test/fetch/xhr_abort.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 The Emscripten Authors.  All rights reserved.
+// Emscripten is available under two separate licenses, the MIT license and the
+// University of Illinois/NCSA Open Source License.  Both these licenses can be
+// found in the LICENSE file.
+
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+#include <emscripten/fetch.h>
+
+void handleSuccess(emscripten_fetch_t *fetch)
+{
+  assert(false && "Should not succeed");
+  emscripten_fetch_close(fetch);
+}
+
+void handleError(emscripten_fetch_t *fetch)
```

Comment:
We try to use the llvm coding style which puts opening braces on the same like as the function declaration (same for if/while/for blocks).